libs/vm/include/vm/common.hpp

```diff
@@ -235,7 +236,9 @@ template <typename T>
 class Ptr;
 class Object;
 
-using ChargeAmount = uint64_t;
+using ChargeAmount                           = uint64_t;
+static constexpr ChargeAmount CHARGE_UNIT    = 1u;
```

Comment:
From our previous discussions this is the conversion factor from microseconds of computation to charge right? If so lets call it `COMPUTE_CHARGE_COST` or something along those lines. Maybe even `USEC_COMPUTE_COST`